stdlib/2and3/decimal.pyi

```diff
@@ -209,94 +213,97 @@ class Context:
-    def Etop(self): ...
-    def create_decimal(self, num=...): ...
-    def create_decimal_from_float(self, f): ...
+    def Etiny(self) -> int: ...
+    def Etop(self) -> int: ...
+    def create_decimal(self, num: _DecimalNew = ...) -> Decimal: ...
+    def create_decimal_from_float(self, f: float) -> Decimal: ...
     def abs(self, a: _Decimal) -> Decimal: ...
     def add(self, a: _Decimal, b: _Decimal) -> Decimal: ...
-    def canonical(self, a): ...
-    def compare(self, a, b): ...
-    def compare_signal(self, a, b): ...
-    def compare_total(self, a, b): ...
-    def compare_total_mag(self, a, b): ...
-    def copy_abs(self, a): ...
-    def copy_decimal(self, a): ...
-    def copy_negate(self, a): ...
-    def copy_sign(self, a, b): ...
-    def divide(self, a, b): ...
-    def divide_int(self, a, b): ...
-    def divmod(self, a, b): ...
-    def exp(self, a): ...
-    def fma(self, a, b, c): ...
-    def is_canonical(self, a): ...
-    def is_finite(self, a): ...
-    def is_infinite(self, a): ...
-    def is_nan(self, a): ...
-    def is_normal(self, a): ...
-    def is_qnan(self, a): ...
-    def is_signed(self, a): ...
-    def is_snan(self, a): ...
-    def is_subnormal(self, a): ...
-    def is_zero(self, a): ...
-    def ln(self, a): ...
-    def log10(self, a): ...
-    def logb(self, a): ...
-    def logical_and(self, a, b): ...
-    def logical_invert(self, a): ...
-    def logical_or(self, a, b): ...
-    def logical_xor(self, a, b): ...
-    def max(self, a, b): ...
-    def max_mag(self, a, b): ...
-    def min(self, a, b): ...
-    def min_mag(self, a, b): ...
-    def minus(self, a): ...
-    def multiply(self, a, b): ...
-    def next_minus(self, a): ...
-    def next_plus(self, a): ...
-    def next_toward(self, a, b): ...
-    def normalize(self, a): ...
-    def number_class(self, a): ...
-    def plus(self, a): ...
+    def canonical(self, a: Decimal) -> Decimal: ...
+    def compare(self, a: _Decimal, b: _Decimal) -> Decimal: ...
+    def compare_signal(self, a: _Decimal, b: _Decimal) -> Decimal: ...
+    def compare_total(self, a: _Decimal, b: _Decimal) -> Decimal: ...
+    def compare_total_mag(self, a: _Decimal, b: _Decimal) -> Decimal: ...
+    def copy_abs(self, a: _Decimal) -> Decimal: ...
+    def copy_decimal(self, a: _Decimal) -> Decimal: ...
+    def copy_negate(self, a: _Decimal) -> Decimal: ...
+    def copy_sign(self, a: _Decimal, b: _Decimal) -> Decimal: ...
+    def divide(self, a: _Decimal, b: _Decimal) -> Decimal: ...
+    def divide_int(self, a: _Decimal, b: _Decimal) -> Decimal: ...
+    def divmod(self, a: _Decimal, b: _Decimal) -> Decimal: ...
+    def exp(self, a: _Decimal) -> Decimal: ...
+    def fma(self, a: _Decimal, b: _Decimal, c: _Decimal) -> Decimal: ...
+    def is_canonical(self, a: _Decimal) -> Decimal: ...
+    def is_finite(self, a: _Decimal) -> Decimal: ...
+    def is_infinite(self, a: _Decimal) -> Decimal: ...
+    def is_nan(self, a: _Decimal) -> Decimal: ...
```

Comment:
Most of these return `bool`, although I haven't checked all of those. I'm guessing `divmod` returns a `Tuple[Decimal, Decimal]`.

Reply:
Good catch. I am rechecking the return values of all `Context` methods.